docs/adding_software/adding_development_software.md

```diff
@@ -113,11 +132,11 @@ release, then include it in the `easyconfigs/` directory.
 
 #### Checksums
 
-EasyBuild's easyconfig files typically contain [checksums](https://docs.easybuild.io/writing-easyconfig-files/?h=checksums#common_easyconfig_param_sources_checksums)
- as their use is highly recommended. By default, EasyBuild will compute the checksums of sources and patch files it needs for
-  a given installation, and compare them with the values in the easyconfig file. Because builds for `dev.eessi.io` change much
-   more often, hard coded checksums become a problem, as they'd need to be updated with every new build. For this reason, we
-    recommend not including checksums in your development easyconfig files (unless you need to, for a specific reason).
+EasyBuild's easyconfig files typically contain [checksums](https://docs.easybuild.io/writing-easyconfig-files/?h=checksums#common_easyconfig_param_sources_checksums) 
+as their use is highly recommended. By default, EasyBuild will compute the checksums of sources and patch files it needs for 
+a given installation, and compare them with the values in the easyconfig file. Because builds for `dev.eessi.io` change much 
+more often, hard coded checksums become a problem, as they'd need to be updated with every new build. For this reason, we 
+recommend not including checksums in your development easyconfig files (unless you need to, for a specific reason).
 
 #### Easystack files 
```

Comment:
For easystack files there is one more gotcha regarding dependencies. The use of `--software-commit` automatically disables the building of dependencies. Including dependencies is still possible but the missing dependencies need to be added to the easystack file *before* the target software.

Most likely you only need to do this for a direct missing dependency (since it is unlikely to use `software-commit` and robot works)

Reply:
Addressed in 4bc63cb